File: src/medium/ArrayStack.swift
func arrayStack(_ target: [Int], _ n: Int) -> [String] {
  var result = [String]()
  var counter = 1
  var index = 0
  while index < target.count {
    result.append("Push")
    if target[index] == counter {
      index += 1
    } else {
      result.append("Pop")
    }
    counter += 1
  }
  return result
}

File: src/main/php/Medium/ArrayStack.php
<?php

declare(strict_types=1);

namespace Src\Medium;

class ArrayStack
{
    /**
     * @param  int[]  $target
     * @return string[]
     */
    public function solve(array $target, int $n): array
    {
        $result = [];
        $index = 0;
        $counter = 1;
        while ($index < count($target)) {
            $result[] = 'Push';
            if ($target[$index] == $counter) {
                $index++;
            } else {
                $result[] = 'Pop';
            }
            $counter++;
        }

        return $result;
    }
}
